Apsitvarkom.UnitTests/Models/Validation/PollutedLocationDTOValidationTests.cs

```diff
@@ -36,8 +36,8 @@ public class PollutedLocationDTOValidationTests
             Notes = "",
             Progress = 0,
             Radius = 0,
-            Severity = "Whatever",
-            Spotted = "timestamp"
+            Severity = "Low",
+            Spotted = "1889-04-20"
```

Comment:
As we've all discussed [here](https://github.com/vu-vibedosa/apsitvarkom-be/issues/4), we'll be using a different type of format - hence we could be focusing on writing tests using similar format.